### ISSUE TITLE
Syntax error in Mongo Driver?

### ISSUE THREAD

**Issue by abhiaiyer91** [meteor/meteor#7929]
Was looking into this piece of code in the Mongo Driver

https://github.com/meteor/meteor/blob/master/packages/mongo/mongo_driver.js#L1092

This syntax isn't correct right? I'm assuming babel is stripping this annotation and the default value of false here is actual not being set.

Does "undefined" here have the same behavior as false?

**Comment by benjamn** [comment]
Ah yes, that only works because the `mongo` package uses `ecmascript` which uses `babel-compiler` which depends on [`meteor-babel`](https://www.npmjs.com/package/meteor-babel) which depends on [`babel-preset-meteor`](https://www.npmjs.com/package/babel-preset-meteor) which [enables the `flow-strip-types` Babel transform](https://github.com/meteor/babel-preset-meteor/blob/e71691ecd754abd79d9d9d3f3b3c1a36648f975c/index.js#L38).

**Comment by benjamn** [comment]
It seems like that method should be written this way:
```js
  count(applySkipLimit = false) {
    return this._synchronousCount(applySkipLimit).wait();
  },
```

**Comment by abernix** [comment]
This is fixed in `devel` thanks to @dr-dimitru in 69ef77bcb6ec8f9982ec6d711fb6c5be997d00bd (#8175).  Thanks for filing this originally, @abhiaiyer91!
